### ISSUE TITLE
Add support for ansible-playbook --check

### ISSUE THREAD

**Issue by JMLX42** [geerlingguy/ansible-role-docker#264]
Hello,

Thank you for the great work!

Before actually running `ansible-playbook` on our production environment, we first run `ansible-playbook --check`. The problem is this role does not work properly in "dry run" (cf https://docs.ansible.com/ansible/2.3/playbooks_checkmode.html).

```
TASK [geerlingguy.docker : Add Docker repository.] *****************************
ok: [build-linux]
ok: [git]
changed: [minecraft]
ok: [vpn-server]
TASK [geerlingguy.docker : Install Docker.] ************************************
ok: [build-linux]
fatal: [minecraft]: FAILED! => {"changed": false, "msg": "No package matching 'docker-ce' is available"}
ok: [git]
ok: [vpn-server]
```

The problems comes from the fact that during a dry run, the `Add Docker repository.` will not *actually* add the apt Docker repository. So the following task, `Install Docker`, fails because the `docker-ce` cannot be found.

The obvious solution is to disable the `Install Docker` task with `when: not ansible_check_mode`. But I'm afraid we would need to do much of the same for the remaining tasks. Which might make it basically mostly useless to run this entire role in dry run mode.

Any idea how to approach/fix this?

Regards,

**Comment by eengstrom** [comment]
I have the same issue and issued a PR (#274) that covers this issue, I believe.

**Comment by JMLX42** [comment]
@geerlingguy any chance to merge #274 ? Thqnk you!

**Comment by geerlingguy** [comment]
Fixed via #274 — thanks!
